### PR TITLE
Added ECR repo ID for Regions

### PR DESCRIPTION
**Pull request by rajkz**
Regions added:

ap-southeast-3
ap-south-2
ap-southeast-4
eu-central-2
eu-south-2
me-central-1

## Description

[Fixes #000.] Fixing Issue mentioned in : (https://github.com/gruntwork-io/kubergrunt/issues/193)

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [ ] Update the docs.
- [ ] Run the relevant tests successfully, including pre-commit checks.
- [ ] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [ ] Include release notes. If this PR is backward incompatible, include a migration guide.

## Release Notes (draft)

Added / Removed / Updated [X].

### CODE REVIEW

**Comment by arsci** [comment]
Thanks for your submission, @rajkz ! I'll kick off automated tests

File: eks/sync.go [hunk 1]
```diff
@@ -76,6 +76,12 @@ var (
 		"me-south-1":     "558608220178",
 		"us-gov-east-1":  "151742754352",
 		"us-gov-west-1":  "013241004608",
+		"ap-southeast-3": "296578399912",
+		"ap-south-2":     "900889452093",
+		"ap-southeast-4": "491585149902",
+		"eu-central-2":   "900612956339",
+		"eu-south-2":     "455263428931",
+		"me-central-1":   "759879836304"
```
Comment (arsci) [inline eks/sync.go:84]: Tests are failing on syntax, I think you might need to add a trailing `,` here.

**Review by arsci — CHANGES_REQUESTED** [review]

**Review by pras111gw — APPROVED** [review]
lgtm

**Comment by pras111gw** [comment]
This commit is merged into issue/201 branch and will be released in next few days.

**Pull request closed without merging**